fem/pfespace.hpp

```diff
@@ -72,6 +72,9 @@ class ParFiniteElementSpace : public FiniteElementSpace
    mutable HypreParMatrix *P;
    /// Optimized action-only prolongation operator for conforming meshes. Owned.
    mutable Operator *Pconf;
+   /// Optimized action-only diagonal prolongation operator on ldofs for
+   /// conforming meshes. Owned.
+   mutable Operator *Pconf_local;
```

Comment:
If I understand correctly, this converts T-dofs to L-dofs, setting any not-owned DOF to zero. Is this an abstraction that makes sense more generally? Is this something that would be useful for users to have access to? It does not seem very intuitive to me.

Reply:
This is how we implement an MFEM `Interpolator` (as opposed to an `Integrator`) with partial assembly. Where in an assembled matrix you use `SetValues` instead of `AddValues`, for matrix-free you have to take care of this in the element restriction (Ceed's `G` operator). I think it makes sense as generally as the `Interpolator` idea.

Reply:
Ok, that makes sense to me. My only concern would be that between `GetConformingProlongation`, `GetProlongationMatrix` and `GetLocalProlongationOperator`, it may be a little bit confusing for users. I think this could be solved with documentation/comments.